images/overview.drawio

```diff
@@ -1 +1,191 @@
+          <mxGeometry x="-0.3488" relative="1" as="geometry">
+            <mxPoint x="420" y="430" as="targetPoint" />
+            <mxPoint as="offset" />
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="w8uqy7LiZfYx2dJa-kWT-5" value="IP address or hostname" style="rounded=0;whiteSpace=wrap;html=1;fontFamily=Noto Sans;fontSource=https%3A%2F%2Ffonts.googleapis.com%2Fcss%3Ffamily%3DNoto%2BSans;" parent="1" vertex="1">
+          <mxGeometry x="20" y="597.5" width="80" height="25" as="geometry" />
+        </mxCell>
+        <mxCell id="w8uqy7LiZfYx2dJa-kWT-7" value="5.2 Well-Known URIs" style="text;html=1;strokeColor=none;fillColor=none;align=left;verticalAlign=middle;whiteSpace=wrap;rounded=0;dashed=1;fontColor=#3399FF;fontFamily=Noto Sans;fontSource=https%3A%2F%2Ffonts.googleapis.com%2Fcss%3Ffamily%3DNoto%2BSans;" parent="1" vertex="1">
+          <mxGeometry y="560" width="180" height="20" as="geometry" />
+        </mxCell>
+        <mxCell id="qLCMJQ3V6eApqyI5j77z-2" value="&lt;div&gt;Thing&lt;br&gt;Link&lt;/div&gt;" style="rounded=1;whiteSpace=wrap;html=1;fontFamily=Noto Sans;fontSource=https://fonts.googleapis.com/css?family=Noto+Sans;" vertex="1" parent="1">
+          <mxGeometry x="540" y="430" width="90" height="55" as="geometry" />
+        </mxCell>
+        <mxCell id="qLCMJQ3V6eApqyI5j77z-4" value="&lt;div&gt;GET&lt;/div&gt;" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;entryX=0;entryY=0.5;entryDx=0;entryDy=0;fontFamily=Noto Sans;fontSource=https%3A%2F%2Ffonts.googleapis.com%2Fcss%3Ffamily%3DNoto%2BSans;exitX=1;exitY=0.77;exitDx=0;exitDy=0;exitPerimeter=0;" edge="1" parent="1" source="Z3I2DZgx9jWYCCeo3zKG-5" target="qLCMJQ3V6eApqyI5j77z-2">
+          <mxGeometry relative="1" as="geometry">
+            <mxPoint x="490" y="408" as="sourcePoint" />
+            <mxPoint x="540" y="400" as="targetPoint" />
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="qLCMJQ3V6eApqyI5j77z-6" value="describedby, hosts, external" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;exitX=1;exitY=0.5;exitDx=0;exitDy=0;fontFamily=Noto Sans;fontSource=https%3A%2F%2Ffonts.googleapis.com%2Fcss%3Ffamily%3DNoto%2BSans;entryX=0.83;entryY=1;entryDx=0;entryDy=0;entryPerimeter=0;" edge="1" parent="1" source="qLCMJQ3V6eApqyI5j77z-2" target="Z3I2DZgx9jWYCCeo3zKG-5">
+          <mxGeometry relative="1" as="geometry">
+            <mxPoint x="455" y="720" as="sourcePoint" />
+            <mxPoint x="490" y="460" as="targetPoint" />
+            <Array as="points">
+              <mxPoint x="700" y="458" />
+              <mxPoint x="700" y="540" />
+              <mxPoint x="465" y="540" />
+            </Array>
+          </mxGeometry>
+        </mxCell>
+      </root>
+    </mxGraphModel>
+  </diagram>
+</mxfile>
```

Comment:
![images/overview.png](https://raw.githubusercontent.com/w3c/wot-discovery/2e0cb1356bf56fa3b8853e523edd904dcfca8a77/images/overview.png)

This figure is confusing:
1. The diagram makes it look like Thing Link and TD are different entities. Thing Link is a type of TD. 
2. Exploration enclosure is covering only directory. Instead it should cover everything after URL is found. Note: "These URLs do not themselves contain metadata, but are used in the second stage, "Exploration", to actually fetch metadata."

I'd instead point from URL to TD (self-described Thing) and use empty arrow heads to divide it using from there into Thing Directory (pointing back to TD) and Thing Link (pointing back to URL).